language/German/strings.po

```diff
@@ -887,18 +893,24 @@ msgid ""
 "One or more ROMs/disk images for this system are incorrect. The system may "
 "not run correctly.\n"
 msgstr ""
+"Ein oder mehrere ROMs/Disk-Images für dieses System sind fehlerhaft. Das "
```

Comment:
better "nicht korrekt" or "inkorrekt" rather than "fehlerhaft". We don't know if they're faulty/erroneous, just that they're not the correct / expected ones

Reply:
There is a ton of other places where _incorrect_ is translated with _fehlerhaft_. I was trying to be consistent here, but _inkorrekt_ it is then